### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/bruno/actions/runs/6342991445/attempts/1#summary-17229905751

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/bruno/actions/runs/6342991568/attempts/1#summary-17229906223

### CODE REVIEW

File: packages/bruno-app/src/components/BrunoSupport/index.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const BrunoSupport = ({ onClose }) => {
           <div className="mt-2">
             <a href="https://github.com/usebruno/bruno" target="_blank" className="flex items-end">
               <IconBrandGithub size={18} strokeWidth={2} />
-              <span className="label ml-2">Github</span>
+              <span className="label ml-2">GitHub</span>
```
Comment (jsoref) [inline packages/bruno-app/src/components/BrunoSupport/index.js:32]: brand -- note that I'm not changing things like `IconBrandGithub` that requires too much thought.

File: packages/bruno-app/src/components/Modal/index.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import StyledWrapper from './StyledWrapper';
 
 const ModalHeader = ({ title, handleCancel }) => (
   <div className="bruno-modal-header">
-    {title ? <div className="bruno-modal-heade-title">{title}</div> : null}
+    {title ? <div className="bruno-modal-header-title">{title}</div> : null}
```
Comment (jsoref) [inline packages/bruno-app/src/components/Modal/index.js:6]: hopefully nothing else cares?

File: packages/bruno-app/src/components/RequestTabs/StyledWrapper.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 
 const Wrapper = styled.div`
-  border-bottom: 1px solid ${(props) => props.theme.requestTabs.borromBorder};
+  border-bottom: 1px solid ${(props) => props.theme.requestTabs.bottomBorder};
```
Comment (jsoref) [inline packages/bruno-app/src/components/RequestTabs/StyledWrapper.js:4]: notable

File: packages/bruno-app/src/components/Sidebar/NewRequest/index.js [hunk 4]
```diff
@@ -5,14 +5,14 @@ import toast from 'react-hot-toast';
 import { uuid } from 'utils/common';
 import Modal from 'components/Modal';
 import { useDispatch } from 'react-redux';
-import { newEphermalHttpRequest } from 'providers/ReduxStore/slices/collections';
+import { newEphemeralHttpRequest } from 'providers/ReduxStore/slices/collections';
```
Comment (jsoref) [inline packages/bruno-app/src/components/Sidebar/NewRequest/index.js:8]: I believe this is fine...

**Comment by helloanoop** [comment]
This is great!

Thank you so much @jsoref !

**Pull request merged**